src/confio/proofs.ts

```diff
@@ -900,7 +900,6 @@ export enum HashOp {
   KECCAK = 3,
   RIPEMD160 = 4,
   BITCOIN = 5,
-  SHA512_256 = 6,
```

Comment:
did this have to be manually removed or did it come along with the automation?

Reply:
This come from automation.